### PR TITLE
Grafana update

### LINKED ISSUE #737: Document using Grafana Cloud with InfluxDB 2.0
The Grafana guide currently only covers using the open source version of Grafana with InfluxDB 2.0, but users should be able to use Grafana Cloud as well. The biggest difference that I'm aware of is how the Flux data source plugin is installed, but I'm not 100% sure of the process.

##### Relevant URLs
- https://v2.docs.influxdata.com/v2.0/visualize-data/other-tools/grafana/

### LINKED ISSUE #1158: Update Grafana details for 7.1 release
New Grafana release coming July 15. We need to review the images in our docs and retake screenshots as needed. 

- https://v2.docs.influxdata.com/v2.0/visualize-data/other-tools/grafana/

### PR DESCRIPTION
**Pull request by sanderson**
Closes #1158
Closes #737

DO NOT MERGE until Grafana 7.1 is released (expected July 15th)

With Grafana 7.1 and Grafana Cloud, Flux support is built directly into the InfluxDB integration. This PR updates the workflow for using Grafana *and* Grafana Cloud with InfluxDB OSS and InfluxDB Cloud.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/visualize-data/other-tools/grafana.md [hunk 1]
```diff
@@ -11,62 +10,48 @@ weight: 201
...
-7. Select **Flux (InfluxDB) [BETA]** from the list of available plugins.
+7. Select **InfluxDB** from the list of available data sources.
 
 ## Configure your InfluxDB 2.0 connection
-1. Enter a name for your Flux datasource.
-2. Under **HTTP**, enter your [InfluxDB URL](/v2.0/reference/urls/).
-3. Under **Auth**, select **With Credentials**.
-4. Under **InfluxDB 2.0.0 Details**, enter your **organization name** _or_ **organization ID**,
-   **default bucket**, and **authentication token**.
+1. Enter a **name** for your InfluxDB data source.
```
Comment (kelseiv) [inline content/v2.0/visualize-data/other-tools/grafana.md:32]: In the Grafana UI, enter?

File: content/v2.0/visualize-data/other-tools/grafana.md [hunk 2]
```diff
@@ -11,62 +10,50 @@ weight: 201
 v2.0/tags: [grafana]
 ---
 
-Use the [Grafana Flux Data Source plugin](https://grafana.com/grafana/plugins/grafana-influxdb-flux-datasource)
-to connect Grafana to InfluxDB 2.0 or InfluxDB Cloud 2.0.
+Use [Grafana](https://grafana.com/) to visualize data from **InfluxDB 2.0** and **InfluxDB Cloud**.
```
Comment (noramullen1) [inline content/v2.0/visualize-data/other-tools/grafana.md:13]: should this use the cloud name shortcode?

Comment (sanderson) [reply content/v2.0/visualize-data/other-tools/grafana.md:13]: Good call. Updating.

File: content/v2.0/visualize-data/other-tools/grafana.md (hunk 1 above)
Comment (kelseiv) [inline content/v2.0/visualize-data/other-tools/grafana.md]: How do I start to configure a connection? Where do I enter the InfluxDB data source name?

File: content/v2.0/visualize-data/other-tools/grafana.md [hunk 3]
```diff
@@ -1,72 +1,114 @@
...
-  InfluxDB Cloud 2.0.
+  Use [Grafana](https://grafana.com/) to visualize data from **InfluxDB 2.0** and **InfluxDB Cloud**.
 menu:
   v2_0:
     name: Use Grafana
     parent: Other visualization tools
 weight: 201
 v2.0/tags: [grafana]
+related:
+  - https://grafana.com/docs/, Grafana documentation
```
Comment (noramullen1) [inline content/v2.0/visualize-data/other-tools/grafana.md:12]: is this 'grafana documentation' supposed to be in here after the URL?

Comment (sanderson) [reply content/v2.0/visualize-data/other-tools/grafana.md:12]: Yeah, under `related`, you can add external related links. Everything before the comma is the link URL. Everthing after the comma is the link text.

**Pull request merged**